packages/odyssey-react-mui/src/Typography.tsx

```diff
@@ -147,6 +149,7 @@ const Typography = ({
       tabIndex={-1}
       translate={translate}
       variant={typographyVariantMapping[variant]}
+      role={role}
```

Comment:
On the other hand, I don't think we should be adding `role` to every Typography element. It will lead to weird situations.

Having our `Heading` component with a custom `role` of a paragraph would defeat the reason for a `Paragraph` component.

If we want to make any typography field optionally presentational, we should add a boolean prop instead.

Something like `isPresentational` would be good.

Reply:
updated